Tests/js/views/ez-userprofileview.html

```diff
@@ -0,0 +1,80 @@
+<!doctype html>
+<html>
+<head>
+<title>eZ User Profile view tests</title>
+</head>
+<body>
+<div class="container"></div>
+
+
+<script type="text/x-handlebars-template" id="userprofileview-ez-template">
+    <div class="ez-user-profile">
+        <div class="ez-user-profile-info">
+            <p class="ez-user-profile-name">{{user.name}}</p>
+        </div>
+        <img class="ez-user-profile-image" src="{{#if user.avatar.uri}}{{user.avatar.uri}}{{else}}{{defaultAvatarImage}}{{/if}}" alt="{{avatarAltText}}">
+    </div>
+</script>
+
+<script type="text/x-handlebars-template" id="usermenuitemfireeventview-ez-template">
+    <a class="ez-user-menu-item">{{ title }}</a>
+</script>
+
+<script type="text/x-handlebars-template" id="usermenuitemview-ez-template">
+    <a class="ez-user-menu-item">{{ title }}</a>
+</script>
+
+<script type="text/javascript" src="../../../Resources/public/vendors/yui3/build/yui/yui.js"></script>
+<script type="text/javascript" src="./assets/ez-userprofileview-tests.js"></script>
+<script>
+    var filter = (window.location.search.match(/[?&]filter=([^&]+)/) || [])[1] || 'raw',
+        loaderFilter;
+
+    if (filter == 'coverage'){
+        loaderFilter = {
+            searchExp : "/Resources/public/js/",
+            replaceStr: "/Tests/instrument/Resources/public/js/"
+        };
+    } else {
+        loaderFilter = filter;
+    }
+
+    YUI({
+        coverage: ['ez-userprofileview'],
+        filter: loaderFilter,
+        modules: {
+            "ez-userprofileview": {
+                requires: ['ez-templatebasedview', 'event-tap', 'ez-usermenuview'],
+                fullpath: "../../../Resources/public/js/views/ez-userprofileview.js"
+            },
+            "ez-usermenuview": {
```

Comment:
same remark as in https://github.com/ezsystems/PlatformUIBundle/pull/513/files#r53777165, you don't need `ez-usermenuview` to test the UserProfileView as long as you inject a view (or a mock) with a `toggleDisplayed` method.